Userland/Libraries/LibCore/LocalServer.cpp

```diff
@@ -146,6 +146,7 @@ RefPtr<LocalSocket> LocalServer::accept()
         perror("accept");
         return nullptr;
     }
+    fcntl(accepted_fd, F_SETFD, FD_CLOEXEC);
```

Comment:
It would be nicer if we had something like `accept4()` that can take `SOCK_CLOEXEC`.

Reply:
It certainly would be nice, I'll have a look. :)